### PR TITLE
Graph optimization + some bugfixes

### PR DESCRIPTION
**Pull request by JulianWielga**

### CODE REVIEW

File: ui/client/components/graph/Graph.js [hunk 1]
```diff
@@ -1,22 +1,21 @@
...
 import "../../stylesheets/graph.styl"
 import "./svg-export/export.styl"
-import * as EspNode from "./EspNode"
 import * as GraphUtils from "./GraphUtils"
 import * as JointJsGraphUtils from "./JointJsGraphUtils"
 import EdgeDetailsModal from "./node-modal/EdgeDetailsModal"
 import NodeDetailsModal from "./node-modal/NodeDetailsModal"
 import NodeUtils from "./NodeUtils"
 import {prepareSvg} from "./svg-export/prepareSvg"
+import {drawGraph, directedLayout, isBackgroundObject, createPaper} from "./GraphPartialsInTS"
```
Comment (JulianWielga) [inline ui/client/components/graph/Graph.js:18]: I know this is an unusual solution, but I didn't want to rewrite (yet) the entire Graph module to ts.

File: ui/client/components/graph/EspNode/element.ts [hunk 2]
```diff
@@ -61,106 +61,68 @@ function getBodyContent(node) {
...
-  const hasErrors = hasCounts && processCounts && processCounts.errors > 0
-  const countsContent = hasCounts ? processCounts ? `${processCounts.all}` : "0" : ""
-  let extraDigitsCount = Math.max(countsContent.length - breakPoint, 0)
-  extraDigitsCount = Math.min(extraDigitsCount, maxExtraDigits)
-
-  return {
-    text: countsContent,
-    fill: hasErrors ? "red" : "#CCCCCC",
-    refX: width - testResultsWidth / 2,
-    // magic/hack: central vertical position when font-size changes
```
Comment (mproch) [inline ui/client/components/graph/EspNode/element.ts:77]: Why is not needed not?

File: ui/client/components/graph/EspNode/esp.ts [hunk 3]
```diff
@@ -1,134 +1,213 @@
 /* eslint-disable i18next/no-literal-string */
```
Comment (lciolecki) [inline ui/client/components/graph/EspNode/esp.ts:1]: I think esp is not good enough name.. What kind of elements we keep there? Configs / styles?

File: ui/client/components/graph/GraphPartialsInTS/drawGraph.ts [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+export function drawGraph(
+  process: Process,
+  layout: Layout,
+  processCounts: ProcessCounts,
+  processDefinitionData: ProcessDefinitionData,
+  expandedGroups: GroupId[],
+) {
+  const graph: dia.Graph = this.graph
+  const directedLayout = this.directedLayout
+  const _updateLayout = updateLayout(graph, directedLayout)
```
Comment (lciolecki) [inline ui/client/components/graph/GraphPartialsInTS/drawGraph.ts:21]: Why these twoo are with underscore.. ?

**Pull request merged**